fastlane/README.md

```diff
@@ -29,6 +29,14 @@ Fetches and updates certificates and provisioning profiles for App Store distrib
 
 Fetches and updates certificates and provisioning profiles for Ad-Hoc distribution
 
+### sync_signing_alpha
+
+```sh
+[bundle exec] fastlane sync_signing_alpha
+```
+
```

Comment:
It's probably obvious what it does, but the spacing makes it seem like the intention was to have a comment describing what it does here

Reply:
It was… I will push that